### ISSUE TITLE
QOS: ensurance analyzer: Reset the element of triggered map duplicately when compute the action context

### ISSUE THREAD

**Issue by shanyuyulaifengmanlou** [gocrane/crane#637]
**Describe the bug**
the demo is here  : pkg/ensurance/analyzer/analyzer.go:286
![image](https://user-images.githubusercontent.com/44322290/204979933-e10043aa-4b09-4186-b9dc-7c78deb6be60.png)

**Reproduce steps**
1. analyze the node QOS (u might call it ensurance):
2. if aboveThreshold is true —— it is undoubtfully to reset restored map where key = nodeName + ruleName. it means the Restore Process(please allow me to call it) is broken off once the aboveThreshold is true.
3. if aboveThreshold is false —— it might mean the node is in the <Restore Processing> . In the future,  the node would be judged as <Restored = true> when the restore times accumulate to object.RestoreThreshold. At that time, the element of triggered map would be reset to zero , meaning the node could be forgiven and be A restored node again **ONLY IF THE CONDITION (s.triggered[key] >= uint64(object.AvoidanceThreshold) &&  s.restored[key] >= uint64(object.RestoreThreshold)) IS TRUE.**
In summary , i think the demo in (pkg/ensurance/analyzer/analyzer.go:286) should be deleted, meanwhile the demo in (pkg/ensurance/analyzer/analyzer.go:298) remains.

**Expected behavior**
the duplicated reset step would lead to the condition (pkg/ensurance/analyzer/analyzer.go:295) be meaningless forever.
**Screenshots**
![image](https://user-images.githubusercontent.com/44322290/204982259-07bbd910-05a1-46f7-9aea-53f63d56a4c0.png)

**Environment (please complete the following information):**
 - K8S Version: [e.g. 1.19]
 - Crane Version: [e.g. 0.1.0]

Sincerely.

**Comment by shanyuyulaifengmanlou** [comment]
Correct: reset restored map where key = nodeName + ruleName ==> key =  nodeQoSName + ruleName

**Comment by chenkaiyue** [comment]
Thx for your issue, can you create a PR?

**Comment by shanyuyulaifengmanlou** [comment]
I am glad to hear that, this is my pr https://github.com/gocrane/crane/pull/642
thanks!
